packages/website/src/building-blocs/PropsTable.tsx

```diff
@@ -0,0 +1,69 @@
+import {ComponentMetadata} from '@coveord/plasma-components-props-analyzer';
+import {FunctionComponent} from 'react';
+
+export interface PropsTableProps {
+    propsMetadata?: ComponentMetadata[];
+}
+
+export const PropsDoc: FunctionComponent<PropsTableProps> = ({propsMetadata = []}) => {
+    if (propsMetadata.length < 1) {
+        return <p>This component has no props.</p>;
+    }
+
+    return (
+        <div className="props-doc">
+            <table className="full-content-x">
+                <thead className="body-m">
+                    <tr>
+                        <th>Name</th>
+                        <th>Type</th>
+                        <th>Default</th>
+                        <th>Description</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    {propsMetadata
+                        .sort(requiredFirst)
+                        .map(({name, type, description, optional, deprecation, defaultValue, params}) => (
+                            <tr key={name}>
+                                <td>
+                                    <span className="code">{name}</span>
+                                    {optional ? null : <span className="body-s-book-italic-subdued ml2">required</span>}
+                                    {deprecation !== null ? (
+                                        <span className="body-s-book-italic-subdued ml2">deprecated</span>
+                                    ) : null}
+                                </td>
+                                <td>
+                                    <span className="code">{type}</span>
+                                </td>
+                                <td>{defaultValue}</td>
+                                <td className="py1">
+                                    {deprecation !== null && <div>{deprecation}</div>}
```

Comment:
deprecated at line 32 and here? 👀

Reply:
I don't understand your question 🙈 Oh and btw I didn't change this code, I just renamed the file and remove a bunch of unneeded stuff

Reply:
No, the first one write `deprecated` next to the prop name. The second writes the deprecation notice in the description column

```ts
    /**
     * @deprecated do not use
     */
    itemFilterLabel?: string;
    /**
     * @deprecated do not use
     */
    itemTooltipProps?: ITooltipProps;
    /**
     * @deprecated do not use
     */
    onClearItemFilter?: () => void;
    /**
     * @deprecated do not use
     */
    itemFilterCropLength?: number;
```
<img width="1395" alt="image" src="https://user-images.githubusercontent.com/35579930/200605732-381150d4-0318-46b8-9c89-59390035301a.png">